### PR TITLE
Refresh the Publishing@W3C pages

### PR DESCRIPTION
**Pull request by iherman**
Changes:
- commented out the "Meeting Summaries" box on the right (if the habit of meeting summaries comes back, we can reinstate it)
- refreshed the list of groups and resources on the left-hand navigation column
- made minor changes on the text right before the "How to Participate" header
- made lots of re-write in the section headed by "How to Participate", trying to give an overview of what we do now.

Comments, questions:

- I was tempted to rename the "Publishing WG" to "Audiobooks WG", but we shouldn't to that until the new charter comes into effect. But, I believe, this should be done eventually
- The testimonial collection comes from the IDPF merger, I wonder whether we should still keep them. If we decide to remove them, we may want to remove the complete right column altogether

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/publishing/pull/22.html" title="Last updated on Aug 27, 2020, 4:56 AM UTC (57804f9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/publishing/22/1490f9e...57804f9.html" title="Last updated on Aug 27, 2020, 4:56 AM UTC (57804f9)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
@TzviyaSiegman @swickr is it o.k. if I merge this? The current page is hopelessly outdated, it would be good to get this done (we can still change it later).

**Comment by TzviyaSiegman** [comment]
sorry, working my way through the backlog. I have some minor edits. I'll add them today

File: index.html [hunk 1]
```diff
@@ -117,30 +117,41 @@ <h1 class="title">Publishing@W3C</h1>
...
+                                </p>
+
+                                <p>
+                                    The EPUB 3 Working Group is closely related to the <a href="http://www.w3.org/community/publishingcg/">Publishing Community Group</a>. While the EPUB 3 Working Group is the guardian of the EPUB standard, the Community Group is the home for discussions around new technical directions, ideas, and experimentation; some of these ideas may become part of a future EPUB or other publishing standards. It is the primary “incubation arm” or the of Publishing@W3C, the birthplace of new ideas for the future. We encourage you to join the new <a href="http://www.w3.org/community/publishingcg/">EPUB 3 Community Group</a> which is free and open to all. Simply go there to sign up.
```
Comment (TzviyaSiegman) [inline index.html:142]: I don't seem to be able to edit directly here. one minor change:
s/ some of these ideas may become part of a future EPUB or other publishing standards. It is the primary “incubation arm” or the of Publishing@W3C/
some of these ideas may become part of a future EPUB or other standards. It is the primary “incubation arm” or the of Publishing@W3C

Comment (iherman) [reply index.html]: Thanks, done it!

**Comment by iherman** [comment]
Got a green light from @swickr to merge...

**Pull request merged**